README.md

```diff
@@ -207,6 +207,31 @@ await DdSdkReactNative.initialize(config);
 
 {{< /site-region >}}
 
+### Overriding the reported version
+
+By default, the SDK will report the `version` as the commercial version of your app (e.g. "1.2.44").
+
+If you use an Over The Air (OTA) updates provider like Microsoft's Codepush, you can override this version to indicate which version of your javascript code is running.
+
+The recommended way is to do so by specifying a `versionSuffix` to the `DdSdkReactNativeConfiguration` object:
+
+```js
+const config = new DdSdkReactNativeConfiguration(
+    '<CLIENT_TOKEN>',
+    '<ENVIRONMENT_NAME>',
+    '<RUM_APPLICATION_ID>',
+    true,
+    true,
+    true
+);
+
+config.versionSuffix = ':ota-3';
```

Comment:
Wouldn't it be better/easier for customers to handle the `:` separator ourselves? With the current api, version suffix could be anything (e.g.: `.5` or just `5), which could lead to ambiguous results?
Also maybe we should add in the doc the set of allowed characters for this string?

Reply:
Looking at the tags definition (https://docs.datadoghq.com/getting_started/tagging/#define-tags) this makes sense to me.